sphinxcontrib/websupport/storage/sqlalchemy_db.py

```diff
@@ -51,7 +51,7 @@ def nested_comments(self, username, moderator):
 
         # Filter out all comments that are not moderated yet.
         if not moderator:
-            q = q.filter(Comment.displayed is True)
+            q = q.filter(Comment.displayed == True)  # noqa: E712
```

Comment:
```suggestion
            q = q.filter(Comment.displayed)
```

Unless I'm mistaken this should both be equivalent and please ruff.

Reply:
It works with SQLite, but I could not find any documentation for this behavior, so I am not sure it will work with all database backends.

However, I found another solution which is documented, should work everywhere, and pleases ruff.